packages/tinacms/src/tina-cms.tsx

```diff
@@ -179,11 +200,31 @@ export const TinaCMSProvider2 = ({
   data: object
   /** Your React page component */
   children: () => React.ReactNode
-  /** Point to the local version of GraphQL instead of tina.io */
+  /**
+   * The URL for the GraphQL API.
+   *
+   * When working locally, this should be http://localhost:4001/graphql.
+   *
+   * For Tina Cloud, use https://content.tinajs.io/content/my-client-id/github/my-branch
+   */
+  apiURL: string
+  /**
+   * Point to the local version of GraphQL instead of tina.io
+   *
```

Comment:
```suggestion
   * https://tina.io/docs/tinacms-context/#adding-tina-to-the-sites-frontend
   *
```
as per @logan-anderson 's suggestion. Note: we'll have to update that doc along with this change being published.